src/Surfnet/Stepup/Identity/Event/GssfPossessionProvenAndVerifiedEvent.php

```diff
@@ -115,6 +124,11 @@ public function getAuditLogMetadata()
 
     public static function deserialize(array $data)
     {
+        // BC compatibility for test-environment only (2.8.0, fixed in 2.8.1)
+        if (!isset($data['preferred_locale'])) {
```

Comment:
It's really nice of you to include this piece of code. However, it's very installation dependent, if for example another Stepup user with entirely different locale settings would not benefit from this. If you could inject the `default_locale` into this event..

Reply:
We discussed this IRL, this feature is for the benefit of event replays. In event replay context, no email is sent. So the value of 'preferred_locale' is irrelevant. For the sake of efficiency we've set the value with the Surfnet default.